engine_details/architecture/core_types.rst

```diff
@@ -180,8 +180,10 @@ scripting API.
 |                       |                          | no heap allocations.                                                                  |
 +-----------------------+--------------------------+---------------------------------------------------------------------------------------+
 | |span|                | ``std::span``            | Represents read-only access to a contiguous array without needing to copy any data.   |
-|                       |                          | See `pull request description <https://github.com/godotengine/godot/pull/100293>`__   |
-|                       |                          | for details.                                                                          |
+|                       |                          | Note that ``Span`` is designed to be a high performance API: It does not perform      |
```

Comment:
```suggestion
|                       |                          | Note that ``Span`` is designed to be a high performance API: it does not perform      |
```

Reply:
You reverted this change, was that intentional?

Reply:
When I looked at this, the discussion was already resolved, so I assumed it wasn't relevant after all. Doesn't the sentence after a semicolon begin uppercase if it's a full sentence?